papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/controller/ProjectsController.java

```diff
@@ -172,4 +211,11 @@ private Availability updatePersistedAvailability(Availability availability, Avai
         availability.setPastSprintRemainingTime(Duration.ofMinutes(availabilityDto.getTimeRemaining()));
         return availabilityService.save(availability);
     }
+
+    private boolean isUserRoleAdmin(Long projectId, Long userId) {
```

Comment:
To mamy zaimplementowane w trzech miejscach już <3 Znaczy nie czepiam się - rozumiem, jak do tego doszło

Byłbyś może skłonny ukraść wariant od @kjarosh umiejscowiony w klasie `Project` na branchu z numerkiem bodajże 55?